spanner/spanner_snippets/snippet.go

```diff
@@ -703,6 +760,185 @@ func queryNewTable(ctx context.Context, w io.Writer, client *spanner.Client) err
 	}
 }
 
+func writeToDocumentsTable(ctx context.Context, w io.Writer, client *spanner.Client) error {
+	documentsColumns := []string{"UserId", "DocumentId", "Timestamp", "Contents"}
+	m := []*spanner.Mutation{
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{1, 1, spanner.CommitTimestamp, "Hello World 1"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{1, 2, spanner.CommitTimestamp, "Hello World 2"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{1, 3, spanner.CommitTimestamp, "Hello World 3"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{2, 4, spanner.CommitTimestamp, "Hello World 4"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{2, 5, spanner.CommitTimestamp, "Hello World 5"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{3, 6, spanner.CommitTimestamp, "Hello World 6"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{3, 7, spanner.CommitTimestamp, "Hello World 7"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{3, 8, spanner.CommitTimestamp, "Hello World 8"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{3, 9, spanner.CommitTimestamp, "Hello World 9"}),
+		spanner.InsertOrUpdate("DocumentsWithTimestamp", documentsColumns,
+			[]interface{}{3, 10, spanner.CommitTimestamp, "Hello World 10"}),
```

Comment:
Did you wrap here so that it would show up in docs nicely? It's fine to have long lines in Go, so if the actual values of these structs aren't that important, I'd put them all on their own line. It also makes it easier to scan (you can see the deltas between the lines)

Reply:
+1

Reply:
>> Did you wrap here so that it would show up in docs nicely?
Yes.

The key purpose of this particular example code is to demonstrate how to use the constant "spanner.CommitTimestamp" as the value when inserting into a timestamp column. When included in documentation formatted as a long line, that constant was getting truncated from the initially visible sample code requiring users to scroll right in order to see it.

However I'm happy to make the suggested change if you would prefer it to be formatted that way for consistency.

Reply:
SGTM to leave as-is. Good to avoid horizontal scrolling.